### ISSUE TITLE
Format geometry

### ISSUE THREAD

**Issue by TERMEPhilippe** [atolcd/pentaho-gis-plugins#15]
Pentaho 7.1 et Plugin-gis 1.2
En ouvrant des transformations migrées  de la version 5.4, le format de type geometry n'existe plus, notamment dans les steps Altération Structure Flux

**Comment by darbonc** [comment]
Effectivement, la dernière version livrée comporte une régression qui impacte certaines briques dont celles que vous mentionnez. Nous analysons au plus vite l'origine de ce dysfonctionnement pour proposer une nouvelle version. Merci de ce retour

**Comment by darbonc** [comment]
La version 1.2.1 corrige normalement la régression constatée en 1.2.0. Le type "Geometry" redevient accessible. Merci de confirmer.

**Comment by TERMEPhilippe** [comment]
Exact, le type Geometry est à nouveau disponible et fonctionnel. Merci

**Comment by roboguy** [comment]
Pardon me, can @TERMEPhilippe  or @darbonc please run the KTR attached ? I would like to know if this situation works for you. Excusez-moi, Je ne parle pas très bien le français

[read-csv.zip](https://github.com/atolcd/pentaho-gis-plugins/files/1195038/read-csv.zip)

**Comment by darbonc** [comment]
Its OK for me with 1.2.1 version

![read_csv_darbonc](https://user-images.githubusercontent.com/13120501/28908890-5bfe1464-7826-11e7-90dc-436789fb6fbf.png)

**Comment by roboguy** [comment]
Thank you, I see you have kept the `GeometryInterface`. That should be enough to allow other plugins to interface with yours as in the example transformation. I wish there was an easier way to add a datatype.

I had submitted a pull request that uses `GeometryInterface` in all plugins & updates some deprecated methods. I would like to rebase and resubmit for your consideration.
